bodhi/server/scripts/approve_testing.py

```diff
@@ -100,14 +121,19 @@ def main(argv=sys.argv):
             # not reached the karma threshold.
             if update.meets_testing_requirements:
                 print(f'{update.alias} now meets testing requirements')
-                text = str(
-                    config.get('testing_approval_msg') % update.mandatory_days_in_testing)
-                update.comment(db, text, author=u'bodhi')
-
-                notifications.publish(
-                    topic='update.requirements_met.stable',
-                    msg=dict(update=update))
-                db.commit()
+                if update.autopush:
+                    update_to_stable(update, db)
+                else:
```

Comment:
I don't think we want to if/else this - we want both to happen. Note that we also want to ensure that the update actually reached the number of days in testing here before pushing.

Reply:
> Note that we also want to ensure that the update actually reached the number of days in testing here before pushing.

Would not that been check by  `meets_testing_requirements` ?

Reply:
Just to make sure I understand this correctly, We want to add a comment to the update to inform the packager this can be pushed to stable, and then if `autopush` is true we push it ? Correct ?